README.md

```diff
@@ -1,6 +1,193 @@
+
+For testing, all XSpec scenarios (`*.xspec`) can be run in place to generate local test reports.
+
+Users are also expected to call resources in this repository from their own scripts. Do this either by cloning, copying and modifying scripts here; by writing your own; or by adapting code into the XML/XSLT processing framework or stack of your choice.
+
+A convention is used indicating that an XProc (`*.xpl` file) or XSLT (`*.xsl`) intended to be invoked directly (that is, not only to be used as a module or component) is given a name entirely or partly in `ALL-CAPITALS`. For example, `src/schema-gen/METASCHEMA-ALL-SCHEMAS.xpl` is such an XProc pipeline (a step definition intended to be used directly).
+
+### Dependencies
+
+As noted, the Saxon XSLT engine can be regarded as a *de facto* dependency - while this XSLT-conformant code should in principle run in any processor implementing the language. SaxonHE can be bundled using Maven or another Java packaging technology.
+
+The [POM file](support/pom.xml) for Java/Maven configuration indicates the current tested version of Saxon. At time of writing, Saxon versions 10 and 11 are known to work with this codebase. When reporting bugs please include the version of your processor.
+
+Developers interested in demonstrating the viability of these processes in different processors and environments are eagerly invited to participate in development of this tool or [related tools](#Related_projects).
+
+### Git Client Setup
+
+See more on git setup on the page on [Contributing](CONTRIBUTING).
+
+Clone the project:
+
+```
+git clone --recurse-submodules https://github.com/usnistgov/metaschema-xslt.git
+```
+
+Or to clone with submodule initialization:
+
+```
+git clone --recurse-submodules https://github.com/usnistgov/metaschema-xslt.git
+```
+
+### Initializing Git submodules
+
+This GitHub repository makes use of Git submodules to mount other repositories as subdirectories. When cloning this repo for the first time, you need to initialize these.
+
+```
+git submodule update --init
+```
+
+Or do this in one step using `clone --recurse-submodules` as noted.
+
+### Updating submodules
+
+Submodule contents will be periodically updated. To ensure you have the latest commits for a configured submodule:
+
+```
+git submodule update --init --recursive
+```
+
+## Contact information
+
+Principal Investigator: Wendell Piez, NIST (US National Institute of Standards and Technology) ITL/CSD (Information Technology Laboratory, Computer Security Division).
+
+Email w e n d e l l (dot) p i e z (at) n i s t (dot) g o v.
+
+This initiative spins off from and supports the ITL/CSD [Metaschema](https://pages.nist.gov/metaschema) Project, originally and currently being developed in support of OSCAL, the [Open Security Controls Assessment Language](https://pages.nist.gov/OSCAL).
```

Comment:
An identical or very similar statement already exists above. Suggest remove it from here or re-write it to say that any issue or question related to the MIMF or OSCAL should be posted on the respective GitHub repos.